_documentation/iconography.md

```diff
@@ -39,57 +39,80 @@ layout: documentation
                 </tr>
                 <tr>
                     <th>
-                        Highlight
-                        <div class="hidden-sm-up"><code class="clr-code">.icon-color-highlight</code></div>
+                        <div class="hidden-xs-down">
+                            Highlight
+                        </div>
+                        <div class="hidden-sm-up"><code class="clr-code">.is-highlight <br/> .is-blue</code></div>
```

Comment:
I don't think `br` works on Firefox (http://stackoverflow.com/questions/10500551/br-not-working-in-firefox). You should test this out. If you don't want to test best thing to do is wrap them in two separate divs